### PR TITLE
Add EMQX to Internet of Things

### PR DESCRIPTION
**Pull request by CrazyWisdom**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1106,6 +1106,7 @@ A [human resources management system](https://en.wikipedia.org/wiki/Human_resour
 
 - [DeviceHive](https://www.devicehive.com/) - Open Source IoT Platform with a wide range of integration options. ([Demo](https://playground.devicehive.com/), [Source Code](https://github.com/devicehive/devicehive-java-server)) `Apache-2.0` `Java`
 - [Domoticz](https://www.domoticz.com/) - Home Automation System that lets you monitor and configure various devices like: Lights, Switches, various sensors/meters like Temperature, Rain, Wind, UV, Electra, Gas, Water and much more. ([Source Code](https://github.com/domoticz/domoticz), [Clients](https://github.com/domoticz/domoticz-android)) `GPL-3.0` `C/C++`
+- [EMQX](https://www.emqx.io/) - An ultra-scalable open-source MQTT broker. Connect 100M+ IoT devices in one single cluster, move and process real-time IoT data with 1M msg/s throughput at 1ms latency. ([Demo](https://www.emqx.com/en/mqtt/public-mqtt5-broker), [Source Code](https://github.com/emqx/emqx)) `Apache-2.0` `Erlang`
```
[the remark this answers is not in the document]
Comment (CrazyWisdom) [reply README.md:1109]: > https://www.emqx.io/docs/en/v5.0/deploy/install.html#installation

Thanks, but EMQX also supports Linux, macOS, and Windows. https://www.emqx.io/docs/en/v5.0/deploy/install.html#tgz-linux-macos-windows

So do I still have to modify it?

[the remark this answers is not in the document]
Comment (KieranRobson) [reply README.md:1109]: As Docker is the only language supported on awesome-selfhosted - see: https://n8225.github.io/, you will still have to change it

[the remark this answers is not in the document]
Comment (KieranRobson) [reply README.md:1109]: I didnt see those before haha, Is erlang required to run those? Im not familiar with erlang / tgz

[the remark this answers is not in the document]
Comment (CrazyWisdom) [reply README.md:1109]: > I didnt see those before haha, Is erlang required to run those? Im not familiar with erlang / tgz

No, the Erlang runtime environment is already included in the tgz package.

[the remark this answers is not in the document]
Comment (KieranRobson) [reply README.md:1109]: Unless Erlang is needed to run the software on the server for any of the installation methods then just docker is needed as its the only supported platform on the awesome-self hosted list see https://n8225.github.io/

[the remark this answers is not in the document]
Comment (CrazyWisdom) [reply README.md]: > Unless Erlang is needed to run the software on the server for any of the installation methods then just docker is needed as its the only supported platform on the awesome-self hosted list see https://n8225.github.io/ language input box.

Fixed, thanks!

[the remark this answers is not in the document]
Comment (CrazyWisdom) [reply README.md]: > Unless Erlang is needed to run the software on the server for any of the installation methods then just docker is needed as its the only supported platform on the awesome-self hosted list see https://n8225.github.io/ language input box.

Hi KieranRobson, I have updated the language tag to `Docker`. Is there anything else that needs to be updated?

Thanks!

**Review by KieranRobson — APPROVED** [review]
LGTM

**Review by nodiscc — APPROVED** [review]

**Pull request merged**